build-infrastructure/release-pipeline-stack.yml

```diff
@@ -128,21 +132,21 @@ Resources:
               - Sid: CodeBuildCreateReportAccess
                 Effect: Allow
                 Resource:
-                  - !Sub "arn:aws:codebuild:${AWS::Region}:${AWS::AccountId}:report-group/${BuildCodeBuildProjectName}-*"
+                  - !Sub 'arn:aws:codebuild:${AWS::Region}:${AWS::AccountId}:report-group/${AmdBuildCodeBuildProjectName}-*'
                 Action:
                   - codebuild:CreateReportGroup
                   - codebuild:CreateReport
                   - codebuild:UpdateReport
                   - codebuild:BatchPutTestCases
                   - codebuild:BatchPutCodeCoverages
 
-  BuildCodeBuildProject:
+  AmdBuildCodeBuildProject:
     Type: AWS::CodeBuild::Project
     Properties:
       Artifacts:
         Type: CODEPIPELINE
       ConcurrentBuildLimit: 10
-      Description: A sample agent build on pr
+      Description: A pipeline to build ECS Agent artifacts, sign them, and copy them to the staging bucket
```

Comment:
nit: Is it better to say pipeline stage? Also is this the project that will sign the artifacts too?

Reply:
Oh, this is the description of the whole pipeline so I went with "pipeline" rather than "stage" and yeah, this pipeline will also sign using a CodeBuild project!

Reply:
Oh right! So probably it is a description of BuildAndSignCodePipeline instead?